awx/main/dispatch/worker/base.py

```diff
@@ -148,7 +148,6 @@ def work_loop(self, queue, finished, idx, *args):
             finally:
                 if 'uuid' in body:
                     uuid = body['uuid']
-                    logger.debug('task {} is finished'.format(uuid))
```

Comment:
This is also really noisy, and kind of useless; the dispatcher has _other_ log messages it prints when a job actually succeeds/fails/cancels.